includes/class-sensei-lesson.php

```diff
@@ -4876,7 +4881,7 @@ public static function the_title() {
 
 				<?php
 				/** This filter is documented in includes/class-sensei-messages.php */
```

Comment:
This comment should be moved to where we moved the filter.

Reply:
Thanks! good catch. Updated it here https://github.com/Automattic/sensei/pull/7063/commits/274f15dece0681a08e13a4fd9448f9bbdec8e870